server/utils/nunjucksSetup.ts

```diff
@@ -214,11 +215,13 @@ export function registerNunjucks(app?: express.Express): Environment {
   })
 
   njkEnv.addFilter('dateToUnix', (date: string) => {
-    return moment(date, ['D MMM YYYY', 'YYYY-MM-DD', 'DD/MM/YYYY']).unix()
```

Comment:
Removed additional date formats as should be defined based on usage rather than blanket cover and handle multiple date formats. Adding the multiple formats to this filter was a change made by myself recently but this would signal to anyone using that filter that it can handle any of those dates, possibly at any time which adds a lot of cognitive overhead. So made more specific.